app/controllers/api/irs_attempts_api_controller.rb

```diff
@@ -29,12 +30,32 @@ def create
             headers['X-Payload-Key'] = log_file_record.encrypted_key
             headers['X-Payload-IV'] = log_file_record.iv
 
-            requested_data = s3_helper.s3_client.get_object(
+            response = s3_helper.s3_client.head_object(
               bucket: s3_helper.attempts_bucket_name,
               key: log_file_record.filename,
             )
 
-            send_data requested_data.body.read, disposition: "filename=#{log_file_record.filename}"
+            requested_data_size = response.content_length
+
+            # Maybe this is variable based on the config?
+            buffer_index = 0
+            buffer_size = 100000
```

Comment:
+1 I'd definitely consider making this a config, and we can also use `_` to help break up zeroes, both here and in YAML

```suggestion
            # Maybe this is variable based on the config?
            buffer_index = 0
            buffer_size = 100_000
```

Reply:
Cool! I did not know you could do this.